app/forms/provider_interface/provider_user_invitation_wizard.rb

```diff
@@ -122,11 +127,23 @@ def previous_provider_id
     end
 
     def next_provider_needing_permissions_setup
-      providers.find { |p| provider_permissions.keys.exclude?(p.to_s) }
+      providers.find do |p|
+        provider_permissions.keys.exclude?(p.to_s) ||
+          provider_permissions[p.to_s]['permissions'].reject(&:blank?).blank?
+      end
     end
 
-    def any_provider_needs_permissions_setup
+    def any_provider_needs_permissions_setup?
       next_provider_needing_permissions_setup.present?
     end
+
+    def at_least_one_provider_permission
```

Comment:
This validation works, but I don't think it's required. It should be possible for a user to view applications only for a provider.